internal/providers/expo/history.go

```
@@ -0,0 +1,280 @@
+// Permalinks are capability URLs, no auth travels with the request.
+func FetchServedManifest(ctx context.Context, permalink string) (*ServedManifest, error) {
+	req, err := http.NewRequestWithContext(ctx, http.MethodGet, permalink, nil)
+	if err != nil {
+		return nil, err
+	}
+	req.Header.Set("Accept", "multipart/mixed")
+	resp, err := assetHTTPClient.Do(req)
+	if err != nil {
+		return nil, fmt.Errorf("could not fetch the update manifest: %w", err)
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)